include/nds/arm7/audio.h

```diff
@@ -25,9 +25,10 @@ extern "C" {
 
 #include <nds/arm7/serial.h>
 #include <nds/system.h>
+#include <nds/timers.h>
 
 #define SOUND_VOL(n)        (n)
-#define SOUND_FREQ(n)       ((-0x1000000 / (n)))
+#define SOUND_FREQ(n)       TIMER_FREQ_SHIFT(n, 1, 1)
```

Comment:
This is probably being a bit pedantic, but can you wrap that `n` in parentheses?